integration_tests/src/utils.rs

```diff
@@ -131,6 +135,40 @@ pub fn get_eth_node() -> String {
     format!("http://{}:8545", NODE_IP)
 }
 
+pub fn get_altheadb_contract_addr() -> Address {
+    "0xb9b674D720F96995ca033ec347df080d500c2230"
+        .parse()
+        .unwrap()
+}
+
+pub fn get_test_runner_magic_phone() -> String {
+    "+17040000000".to_string()
+}
+
+pub async fn deploy_contracts() {
+    let contact = Contact::new(
+        &get_althea_grpc(),
+        ALTHEA_CONTACT_TIMEOUT,
+        ALTHEA_CHAIN_PREFIX,
+    )
+    .unwrap();
+    // prevents the node deployer from failing (rarely) when the chain has not
+    // yet produced the next block after submitting each eth address
+    contact.wait_for_next_block(TOTAL_TIMEOUT).await.unwrap();
+
+    let res = Command::new("npx")
+        .args([
+            "ts-node",
+            "/althea_rs/solidity/contract-deployer.ts",
+            &format!("--eth-privkey={}", MINER_PRIVATE_KEY),
+            &format!("--eth-node={}", ETH_NODE),
+        ])
+        .output()
+        .expect("Failed to deploy contracts!");
+
+    info!("Contract deploy returned {:?}", from_utf8(&res.stdout));
```

Comment:
it might be easiest to parse the output here and return the deployed contract address.

Reply:
done